### PR TITLE
Remove final comma in order to be a valid JSON

### PR DESCRIPTION
**Pull request by aarongarciah**

### CODE REVIEW

**Comment by aarongarciah** [comment]
Hi Lea, 

There is a wrong comma in the JSON that prevents Mustache to render.

screen shot 2016-08-30 at 11 20 00

File: data.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
 			"url": "http://intercoolerjs.org/",
 			"image": "https://raw.githubusercontent.com/LeadDyno/intercooler-js/master/www/images/Intercooler_CMYK.png",
 			"description": "AJAX With Attributes: There is no need to be complex",
-			"github": "LeadDyno/intercooler-js",
```
Comment (aarongarciah) [inline data.json:56]: Wrong comma at the end of the line

**Comment by thecristen** [comment]
Good catch, my bad. 😓

**Comment by aarongarciah** [comment]
I noticed because the library list in http://markapp.io is empty  😬

**Comment by LeaVerou** [comment]
Ah, sorry, I fixed that before I saw this PR :(

**Pull request closed without merging**